client/src/components/molecules/AgendaFoldedText.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { Box, Text } from "@/components/atoms";
+
+interface Agenda {
+  title: string;
+  subtitle: string;
+}
+
+interface Props {
+  agenda: Agenda;
+}
```

Comment:
이 부분도 위 코멘트에서 언급한 대로 고치는게 좋을 것 같아요!

```suggestion
interface Props {
  title: string;
  subtitle: string;
}
```